src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -1713,7 +1732,7 @@ import './Menu.css'
 | -- | -- | -- |
 | `role="menu"` | `.pf-c-menu__list` | Declares `.pf-c-menu__list` as a menu. |
 | `disabled` | `button.pf-c-menu__item` | When the menu item uses a button element, indicates that it is unavailable and removes it from keyboard focus. |
-| `role="menuitem"` | `.pf-c-menu__item` | Assigns `.pf-c-menu__item` as an option in a set of choices contained by a menu. |
+| `role="menuitem"` | `.pf-c-menu__item`, `.pf-c-menu__list-item (checkbox)` | Assigns `.pf-c-menu__item` as an option in a set of choices contained by a menu. |
```

Comment:
seems like we would put the checkbox description outside of the backticks, or maybe call it out in the description cell?

```suggestion
| `role="menuitem"` | `.pf-c-menu__item`, `.pf-c-menu__list-item` (checkbox) | Assigns `.pf-c-menu__item` as an option in a set of choices contained by a menu. |
```